doc/rados/configuration/mon-config-ref.rst

```diff
@@ -322,7 +322,7 @@ by setting it in the ``[mon]`` section of the configuration file.
 :Description: Issue a ``HEALTH_WARN`` in cluster log when the monitor's data
               store goes over 15GB.
 :Type: Integer
-:Default: 15*1024*1024*1024*
+:Default: 16106127360
```

Comment:
i think `15*1024*1024*1024` is more readable than 16106127360. one can interpret as 15GB. but probably we can remove the trailing `*` , and quote it with "``".